form/src/components/Result.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+
+const Result = ({ question }) => {
+  const [query, setQuery] = useState("");
```

Comment:
query is not good name for it. Query mostly used for search input texts. Plus, why did you separate image and answer states?